x-pack/plugins/security/server/lib/authorization/privilege_serializer.ts

```diff
@@ -29,8 +30,12 @@ export class PrivilegeSerializer {
     return spaceBasePrivileges.includes(privilegeName);
   }
 
+  public static isSerializedReservedPrivilege(privilegeName: string) {
+    return privilegeName.startsWith(reservedPrefix);
+  }
+
   public static serializeGlobalBasePrivilege(privilegeName: string) {
-    if (!basePrivilegeNames.includes(privilegeName)) {
+    if (!globalBasePrivileges.includes(privilegeName)) {
```

Comment:
Can you explain why we treat `apm`, `ml`, and `monitoring` like global base privileges?

Reply:
This is left-over code that I hacked in the for the product review, good catch, I'll be removing this.